org.mwc.cmap.legacy/src/MWC/GUI/ExtendedCanvasType.java

```diff
@@ -64,5 +64,9 @@ public void semiFillRect(final int x, final int y, final int wid,
 	public void setXORMode(boolean mode);
 	
 	public boolean getXORMode();
+	

```

Comment:
@pecko  - we don't have a requirement for rounded rectangles, not for this issue.  

There was a previous issue where we wanted "fancy" UI, but for this issue, plain rectangular blocks are fine.

So, could you remove these functions from this API, and from SWTCanvasAdapter?  

thx.